CMakeLists.txt

```diff
@@ -495,6 +502,24 @@ endif()
 # initialize CLI11 submodule
 cpp_cc_git_submodule(CLI11 BUILD PACKAGE CLI11 REQUIRED)
 
+# =============================================================================
+# Enable NRN_DIGEST
+# =============================================================================
+if(NRN_ENABLE_DIGEST)
+  set(NRN_DIGEST 1)
+else()
+  set(NRN_DIGEST 0)
+endif()
+
+# =============================================================================
+# Enable NRN_ARCH_INDEP_EXP_POW
+# =============================================================================
+if(NRN_ENABLE_ARCH_INDEP_EXP_POW)
+  set(NRN_ARCH_INDEP_EXP_POW 1)
+else()
+  set(NRN_ARCH_INDEP_EXP_POW 0)
+endif()
+
```

Comment:
It seems the only purpose of these is for a `configure_file`:
```
#cmakedefine NRN_DIGEST @NRN_DIGEST@
```

If that's correct, one could consider using:
```
#cmakedefine01 NRN_DIGEST @NRN_ENABLE_DIGEST@
```
and then removing the extra CMake variables.

See: https://cmake.org/cmake/help/latest/command/configure_file.html#transformations

Reply:
Thanks. That simplifies things a bit.